rs/xnet/payload_builder/tests/certified_slice_pool.rs

```diff
@@ -137,6 +137,8 @@ proptest! {
             if let (Some(byte_limit), Some(prefix)) = (byte_limit, prefix.as_ref()) {
                 assert!(prefix.count_bytes() <= byte_limit);
             }
+            // Testing the signal limit is pointless here because it requires very large streams
+            // that would make this test needlessly slow. There is a dedicated test for it.
```

Comment:
See the comment about making the limits configurable. I could easily add it here but I don't want to generate huge streams all the time.